tests/e2e/test_internal_trades.py

```diff
@@ -15,15 +49,23 @@ def token_slippage(
 
 
 def get_internal_transfers(
-    dune: DuneAnalytics, tx_hash: str, period_start: datetime, period_end: datetime
+    dune: DuneAnalytics,
+    tx_hash: str,
+    period_start: datetime,
+    period_end: datetime,
 ) -> list[InternalTokenTransfer]:
-    path = "./queries/slippage"
-    select_transfers_query = dune.open_query(
-        File("select_in_out_with_buffers.sql", path).filename()
-    )
-    slippage_sub_query = dune.open_query(
-        File("subquery_batchwise_internal_transfers.sql", path).filename()
-    )
+    slippage_sub_query = dune.open_query("./queries/period_slippage.sql")
+    select_transfers_query = """
+        select block_time,
```

Comment:
To be honest, I dislike it.
having all dune queries in one place makes them very easily searchable and understandable. I think the overhead of file-reading for them is negligble

Reply:
Fixed this.